README.md

```diff
@@ -20,7 +20,7 @@ Documentation is available on the [GitHub wiki](https://github.com/microsoft/DxC
 
 ## Notices
 
-All content and source code for this package are subject to the terms of the [MIT License](http://opensource.org/licenses/MIT).
+All content and source code for this package are subject to the terms of the [MIT License](LICENSE).

```

Comment:
This works when clicking on the interface on GitHub, but it's probably better to use ``https://github.com/microsoft/DxCapsViewer/blob/main/LICENSE``